src/trusted/service_runtime/sel_main.c

```diff
@@ -258,17 +258,27 @@ int NaClSelLdrMain(int argc, char **argv) {
   
   cagecount = 0;
 
+  nap = NaClAlignedMalloc(sizeof(*nap), __alignof(struct NaClApp));
+
   /* Initialize cage early on to avoid Cage 0 */
   InitializeCage(nap, 1);
 
+  if (!NaClMutexCtor(&ccmut)) {
+    NaClLog(LOG_FATAL, "%s\n", "Failed to initialize cc mutex");
+  }
+    if (!NaClCondVarCtor(&cccv)) {
+    NaClLog(LOG_FATAL, "%s\n", "Failed to initialize cc cv");
+  }
 
   if (!DynArrayCtor(&nap->children, 16)) {
     NaClLog(1, "%s\n", "Failed to initialize children list");
   }
 
+  LaunchReaper();
```

Comment:
I think calling it a thread reaper rather than a reaper in general is clearer. It makes it clear it isn't an oom reaper or etc.